package com.revature.exceptions.exceptions;

public class UserNotFoundException extens RuntimeException() {
	
	
}